### PR TITLE
[Pages] Replace references of Rules > Bulk Redirects

### PR DESCRIPTION
**Pull request by Erisa**
The Zone level **Rules > Bulk Redirects** does not exist anymore.

It's now Rules > Redirect Rules which takes you to a new page mentioning Bulk Redirects:
image

Rather than go through this, I thought it would be better to go through the main **Account Home > Bulk Redirects**, consistent with the instructions that the main Bulk Redirect documentation uses.

Feedback is welcome! Especially about the image, I wasn't sure if it needed to be cropped smaller or not.

https://erisa-bulk-redirect-account-.cloudflare-docs-7ou.pages.dev/pages/how-to/www-redirect/
https://erisa-bulk-redirect-account-.cloudflare-docs-7ou.pages.dev/pages/how-to/use-bulk-redirects/

### CODE REVIEW

File: content/pages/how-to/use-bulk-redirects.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ For example, if you have Bulk Redirects set up to direct `abc.com` to `xyz.com`
 
 {{</Aside>}}
 
-To use Bulk Redirects, log in to the [Cloudflare dashboard](https://dash.cloudflare.com?to=/:account/:zone/rules) > **Rules** > **Bulk Redirects**.
+To use Bulk Redirects, log in to the [Cloudflare dashboard](https://dash.cloudflare.com?to=/:account/:zone/rules) > **Account Home** > **Bulk Redirects**.
```
Comment (WalshyDev) [inline content/pages/how-to/use-bulk-redirects.md]:
```suggestion
To use Bulk Redirects, log in to the [Cloudflare dashboard](https://dash.cloudflare.com) > **Account Home** > **Bulk Redirects**.
```

Comment (Erisa) [reply content/pages/how-to/use-bulk-redirects.md]: Good catch!

**Comment by WalshyDev** [comment]
> Especially about the image, I wasn't sure if it needed to be cropped smaller or not.

Yeah, this image is pretty huge, cropping would be nice but I understand it does make the screenshot less useful. In theory, the instructions should be enough though

**Comment by Erisa** [comment]
I think a cropped image may not be too bad, the other steps have helpful images and combined with the instructions and links above it, the cropped image should be sufficient 🤔

**Review by deadlypants1973 — APPROVED** [review]

File: content/pages/how-to/use-bulk-redirects.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ For example, if you have Bulk Redirects set up to direct `abc.com` to `xyz.com`
 
 {{</Aside>}}
 
-To use Bulk Redirects, log in to the [Cloudflare dashboard](https://dash.cloudflare.com?to=/:account/:zone/rules) > **Rules** > **Bulk Redirects**.
+To use Bulk Redirects, log in to the [Cloudflare dashboard](https://dash.cloudflare.com) > **Account Home** > **Bulk Redirects**.
 
 ![Bulk redirects option](/pages/how-to/media/bulk_redirects_option.png)
```
Comment (deadlypants1973) [inline content/pages/how-to/use-bulk-redirects.md]: Let's delete this as the menu is clear enough without. Removed the need to update screenshot if UI changes colors or design in next few months.

Comment (deadlypants1973) [inline content/pages/how-to/use-bulk-redirects.md]:
```suggestion
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
closes #7248

**Comment by Erisa** [comment]
Deleted the image since it was removed from the markdown.

Looks good to me now, feel free to merge if you agree @deadlypants1973

**Pull request merged**